src/__tests__/App.test.js

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
```

Comment:
Please write unit tests components and for pure functions in Redux code.
Please write integration tests for app using React Testing Library.

![tests](https://user-images.githubusercontent.com/35504149/206465521-6bfaa859-f52f-429f-9f13-5989dffa7221.png)

Reply:
> Please write unit tests components and for pure functions in Redux code. Please write integration tests for the app using React Testing Library.
> 
> ![tests](https://user-images.githubusercontent.com/35504149/206465521-6bfaa859-f52f-429f-9f13-5989dffa7221.png)

- [ ] As the previous reviewer asked, kindly work on it. :+1: